### DISCUSSION TITLE
Feature request: connected devices

### DISCUSSION THREAD

**Discussion by sebcsaba** [mattlongman/Hassio-Access-Point#40] [Ideas]
I miss a feature to list all connected clients, MAC and IP addresses. This could be very helpful to set up devices connected on this AP to the Hassio.

**Comment by ex-ml** [comment]
As it stands, the addon is quite archaic. It could use a web frontend with ingress to manage the config and devices, plus list devices as you've requested. The current log output could list devices, but other errors/messages would mean you might need to repeat every minute or so.

I'd be happy for someone to add a list of connected devices to the log output.
